pkgs/development/python-modules/pytest-textual-snapshot/default.nix

```diff
@@ -45,6 +46,10 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "pytest_textual_snapshot" ];
 
+  postInstall = ''
+    cp -r resources $out/${python3.sitePackages}/
```

Comment:
Before when snapshot tests fail (for example with `memray`), there was an error that it cannot find `pytest-textual-snapshots/.../resources/...`, no useful info why the tests failed. With this an HTML report gets generated.